contrib/sbase/box_src_generator.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+import argparse
+
+from contextlib import redirect_stdout
+
+
+def generate_core_file(program: str, tools: list):
+    main_prefixes = [tool.replace('-', '_') for tool in tools]
+
+    with open(program + '.c', 'w') as f:
+        with redirect_stdout(f):
+            print('#include <libgen.h>')
+            print('#include <stdio.h>')
+            print('#include <stdlib.h>')
+            print('#include <string.h>')
+            print('#include "util.h"')
+
+            for prefix in main_prefixes:
+                print(f'int {prefix}_main(int, char**);')
+
+            print('int main(int argc, char *argv[]) { \
```

Comment:
Please take into account that the output file should still be readable by human being. It would be better if you could make the output formatted and not using weird constructs .